build.sh

```diff
@@ -19,6 +19,9 @@ elif [ -f "$HOME/Documents/Godot/$GODOT/Godot" ]; then
 # Godot snaps use the "binary-v1-v2" name form
 elif [ -f "/snap/bin/godot-${GODOT%%.*}-${GODOT##*.}" ]; then
     GODOT="/snap/bin/godot-${GODOT%%.*}-${GODOT##*.}"
+# Godot on macos
```

Comment:
Would have been better to capitalize, but still, thanks for this, I don't own a Mac so I could never do this myself.

Reply:
You mean macOS instead of macos?  Sorry, was too lazy but will remember next time ;)
Actually I would love to see UH running on a Mac.